cmd/soroban-cli/src/log/contract_event.rs

```diff
@@ -0,0 +1,19 @@
+pub fn contract_events(events: &[impl std::fmt::Debug], level: tracing::Level) {
```

Comment:
There seems to be three files in this PR that have the exact same contents as this file, just with a different name. Why is that?

Reply:
It had been so that you could filter the tests, but I'll update it to be functions within the main file and it should still work.